static/scss/components/banners.scss

```diff
@@ -86,3 +86,126 @@
         }
     }
 }
+
+// Labels data opt in notification banner
+
+.c-data-notification-banner {
+    display: flex;
+    flex-direction: column;
+    background-color: $color-white;
+    box-shadow: $box-shadow-sm;
+    border-radius: $border-radius-md;
+
+    .data-notification-banner-bg {
+        padding: $spacing-xs;
+        width: 100%;
+        position: relative;
+        min-height: 0;
+    }
+
+    .data-notification-banner-content {
+        padding-left: $spacing-sm;
+        display: flex;
+        flex-direction: row;
+    }
+
+    .data-notification-banner-copy {
+        padding: $spacing-lg;
+    }
+
+    .data-notification-banner-header {
+        font-family: $font-metropolis;
+        font-size: 1.125em;
+        font-weight: 700;
+        text-align: center;
+        width: 90%;
```

Comment:
Is this line necessary? IIRC this looked kinda weird when I tried to use it on the settings page, and it doesn't look any different if I remove it.

Reply:
Hm, I believe I kept it there to make sure it truncated properly at mobile breakpoints (doesn't overlap with the X button)